packages/frontend/src/components/nft/NFTDetail.js

```diff
@@ -0,0 +1,225 @@
+import React, { useEffect, useState } from 'react';
+import { Translate } from 'react-localize-redux';
+import { useSelector } from 'react-redux';
+import styled from 'styled-components';
+
+import { useFungibleTokensIncludingNEAR } from '../../hooks/fungibleTokensIncludingNEAR';
+import UserIconGrey from '../../images/UserIconGrey';
+import { selectAccountId } from '../../redux/slices/account';
+import NonFungibleTokens from '../../services/NonFungibleTokens';
+import FormButton from '../common/FormButton';
+import Container from '../common/styled/Container.css';
+import ArrowIcon from '../svg/ArrowIcon';
+import NFTTransferModal from './NFTTransferModal';
+
+const StyledContainer = styled(Container)`
+    .container {
+        margin-left: auto;
+        margin-right: auto;
+        max-width: 429px;
+        position: relative;
+
+        img {
+            width: 100%;
+            max-width: 429px;
+            margin-bottom: 83px;
+
+            filter: drop-shadow(0px 100px 80px rgba(0, 0, 0, 0.07)) 
+            drop-shadow(0px 41.7776px 33.4221px rgba(0, 0, 0, 0.0503198)) 
+            drop-shadow(0px 22.3363px 17.869px rgba(0, 0, 0, 0.0417275)) 
+            drop-shadow(0px 12.5216px 10.0172px rgba(0, 0, 0, 0.035)) 
+            drop-shadow(0px 6.6501px 5.32008px rgba(0, 0, 0, 0.0282725)) 
+            drop-shadow(0px 2.76726px 2.21381px rgba(0, 0, 0, 0.0196802));
+        }
+
+        .desc {
+            margin-top: 48px;
+            margin-bottom: 56px;
+
+            font-family: Inter;
+            font-style: normal;
+            font-weight: 500;
+            font-size: 16px;
+            line-height: 150%;
+            /* or 24px */
```

Comment:
Can we please remove the reference comments in the CSS? Including `/* gray/neutral/800 */` etc.